home.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Kendralyn Burr - Portfolio</title>
+</head>
+<body>
+  <header>
+      <h1>Kendralyn's Portfolio</h1>
+    <nav>
+      <ul>
+        <li><a href="home.html">Home</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="works.html">Works</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main>
+    <div><img src="images/desktop.jpg" alt="placeholder photo of desktop"></div>
+    <div>
+      <h3 id="section-title">Section-Title</h3>
```

Comment:
You can't skip heading levels. The next one should be h2 and not h3. I am going to merge for now so you can move along and begin your CSS work.

Reply:
Thank you. I realized this was a typo on my part. Thank you for catching it.